### PR TITLE
update placement concept

### PR DESCRIPTION
**Pull request by haoqing0110**
Ref: https://github.com/open-cluster-management/backlog/issues/16066

Signed-off-by: haoqing0110 <qhao@redhat.com>

### CODE REVIEW

File: content/concepts/placement.md [hunk 1]
```diff
@@ -57,7 +61,37 @@ spec:
...
+  - `name` is the name of a prioritizer. Below are the valid names:
+    - Balance: balance the decisions among the clusters.
+    - Steady: ensure the existing decision is stabilized.
+    - ResourceRatioCPU & ResourceRatioMemory: sort clusters based on the allocatable to capacity ratio.
+    - ResourceAllocatableCPU & ResourceAllocatableMemory: sort clusters based on the allocatable.
+  - `weight` defines the weight of prioritizer. The value must be ranged in [0,10].
+    Each prioritizer will calculate an integer score of a cluster in the range of [-100, 100]. The final score of a cluster will be sum(weight * prioritizer_score).
+    A higher weight indicates that the prioritizer weights more in the cluster selection, while 0 weight indicate thats the prioritizer is disabled.
+
+A slice of `PlacementDecision` will be created by placement controller in the same namespace, each with a label of `open-cluster-management.io/placement=placement1`. `PlacementDecision` contains the results of the cluster selection as seen in the following examples.
```
Comment (qiujian16) [inline content/concepts/placement.md:94]: placement.open-cluster-management.io/placement=placement1?

Comment (haoqing0110) [reply content/concepts/placement.md]: Should be `cluster.open-cluster-management.io/placement=placement1` , fixed.

**Pull request merged**